src/rad/appdata.cpp

```diff
@@ -1319,16 +1321,41 @@ bool ApplicationData::LoadProject( const wxString &file, bool justGenerate )
 
 		if ( older )
 		{
+			wxLogDebug("Old version: %d.%d -- current %d.%d", fbpVerMajor, fbpVerMajor, m_fbpVerMajor, m_fbpVerMinor);
+
 			if( justGenerate ){
 				wxLogError( wxT( "This project file is out of date.  Update your .fbp before using --generate" ) );
 				return false;
 			}
-			if ( wxYES == wxMessageBox( wxT( "This project file is not of the current version.\n" )
-			                            wxT( "Would you to attempt automatic conversion?\n\n" )
-			                            wxT( "NOTE: This will modify your project file on disk!" ), _( "Old Version" ), wxYES_NO ) )
+
+			wxFileName fname(file);
+			wxString   strMsg, strOldBackupName(fname.GetFullName() + wxT(".bak"));
+
+			if (wxMessageBox(strMsg.Format(_("This project file is an old version.\n\n"
+			                                 "Do you want to copy the old version to %s and save an updated version?"),
+			                               strOldBackupName),
```

Comment:
Old version sounds so negative, in my opinion not current version or something alike that sounds more friendly. Now that you are changing the message you could also make it more precise, it is not really the file itself that is not current but the file format.

We should use the same terms during the whole operation, here it is called copy, later save and the dialog title says backup. The term backup seems to be fitting best so i think the wording should be like create a backup or use backup as a verb.